__docs__/wonder-blocks-dropdown/action-menu.stories.tsx

```diff
@@ -431,3 +442,91 @@ const locales = [
     {id: "fr", locale: "fr", localName: "français"},
     {id: "it", locale: "it", localName: "italiano"},
 ];
+
```

Comment:
**note:** This story shows how we can use an `ActionMenu` with custom `ActionItem` elements.